### ISSUE TITLE
configuration file

### ISSUE THREAD

**Issue by madzul** [meganz/MEGAcmd#77]
is there any chance to run MEGAcmd using a specific configuration file?

**Comment by polmr** [comment]
There's a configuration file located at `$HOME/.megaCmd/megacmd.cfg` (in linux). Stuff like syncs, backups, webdav locations & other configurations are stored there when modified. At the moment, there's no option whatsoever to change the location of that file.

**Comment by madzul** [comment]
Yes, i'm aware of that. Can we edited the files? like the megacmd.cfg? if so, what options or parameter that we can supplied in the file?

**Comment by polmr** [comment]
`megacmd.cfg` is a text file. You can potentially edit it. But it's preferable that you do everything via MEGAcmd commands (e.g. given, you can set "maxspeedupload" manually before starting megacmd, but it's preferable that you do it via `speedlimit` command). We will however consider providing an skeleton file for manual edition.

**Comment by matthewshear** [comment]
Would it be possible to get some flexibility on the Linux config file? In particular, the ability to specify the location of the config file, the ability to store login credentials like email and password in the config file or possibly the ability to use environment variables for email and password? I'm using MEGAcmd for some shell scripting and I would prefer an alternative to hard coding email and password in the script itself.
